pallets/marketplace/src/lib.rs

```diff
@@ -0,0 +1,512 @@
+#![cfg_attr(not(feature = "std"), no_std)]
+
+pub use pallet::*;
+pub use traits::*;
+pub use types::*;
+
+#[cfg(test)]
+pub mod mock;
+#[cfg(test)]
+mod tests;
+
+// #[cfg(feature = "runtime-benchmarks")]
+// mod benchmarking;
+
+mod traits;
+mod types;
+mod utils;
+pub mod weights;
+
+#[frame_support::pallet]
+pub mod pallet {
+    use core::fmt::Debug;
+
+    use codec::EncodeLike;
+    use frame_support::{
+        dispatch::DispatchResultWithPostInfo, ensure, pallet_prelude::*,
+        sp_runtime::traits::StaticLookup, Blake2_128Concat, PalletId,
+    };
+    use frame_system::pallet_prelude::*;
+    use sp_runtime::traits::{CheckedMul, MaybeDisplay};
+    use sp_std::prelude::*;
+
+    use pallet_acurast::{
+        AllowedSourcesUpdate, Fulfillment, JobHooks, JobId, JobRegistrationFor, Reward, RewardFor,
+        RewardManager, Script, StoredJobRegistration,
+    };
+
+    use crate::traits::*;
+    use crate::types::*;
+    use crate::utils::*;
+
+    #[pallet::config]
+    pub trait Config:
+        frame_system::Config
+        + pallet_timestamp::Config
+        + pallet_assets::Config
```

Comment:
@gitsimon these 2 are not needed and can be removed

Reply:
all warnings such as unused imports are fixed